python/pyspark/mllib/linalg/__init__.py

```diff
@@ -1236,6 +1359,44 @@ def sparse(numRows, numCols, colPtrs, rowIndices, values):
         """
         return SparseMatrix(numRows, numCols, colPtrs, rowIndices, values)
 
+    @staticmethod
+    def fromML(mat):
+        """
+        Convert a matrix from the new mllib-local representation.
+        This does NOT copy the data; it copies references.
+
+        >>> mllibDM1 = Matrices.dense(2, 2, [1, 2, 3, 4])
```

Comment:
Btw, in the future, I'd like us to treat doc tests more like documentation and put unit tests within tests.py.  But this isn't a big deal and I'm OK with leaving your doc tests as they are for now.